DataFormats/Phase2TrackerCluster/interface/Phase2TrackerCluster1D.h

```diff
@@ -0,0 +1,43 @@
+#ifndef DATAFORMATS_PHASE2TRACKERCLUSTER_PHASE2TRACKERCLUSTER1D_H 
+#define DATAFORMATS_PHASE2TRACKERCLUSTER_PHASE2TRACKERCLUSTER1D_H 
+
+#include <stdint.h>
```

Comment:
@pcanal We should move to the fixed width headers. However, I do not know how ROOT interacts with them in classes that are meant to be stored.